source/sink-connector/configuration-properties/cdc.txt

```diff
@@ -45,7 +45,9 @@ Settings
        |
        | **Description:**
        | The class name of the CDC handler to use for converting changes
-         into event streams.
+         into event streams. See
+         :doc:`Available CDC Handlers </sink-connector/fundamentals/change-data-capture#available-cdc-handlers>`
+        for a list of CDC handlers.
```

Comment:
Issue: I think there might be something inconsistent with the indentation on line 50. Did the build log report any warnings/errors?

Suggestion: use appropriate/consistent indentation